bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/gems/GemsVoTableCatalog.scala

```diff
@@ -69,7 +69,7 @@ object GemsVoTableCatalog {
   private def searchCatalog(basePosition: Coordinates, criterions: List[GemsCatalogSearchCriterion], statusLogger: StatusLogger): Future[List[GemsCatalogSearchResults]] = {
     val queryArgs = for {
       c <- criterions
-      q = CatalogQuery(basePosition, c.criterion.radiusLimits, c.criterion.magLimits.some)
+      q = CatalogQuery(0, basePosition, c.criterion.radiusLimits, c.criterion.magConstraints)
```

Comment:
Well, the id is only useful for gems, it is ignored for non-gems, perhaps `Option[Int]` is the right approach here